.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,8 @@
 - [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
+- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
 - [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
 - [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
 - [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
-- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
+- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?
```

Comment:
`--new` isn't a valid flag here

Reply:
I thought `brew audit --new` is valid (I've used it) – we have `--new` and also `--new-formula` and `--new-cask` that are explicit about what the new package is. You have pointed me to a bug, though:
```
➜ brew audit --new-cask appium
Warning: Treating appium as a formula. For the cask, use homebrew/cask/appium
appium:
  * New formulae in homebrew/core should not have a `bottle do` block
Error: 1 problem in 1 formula detected
```

Reply:
I forgot it was valid, ignore me then 😁